actor/src/main/scala/org/apache/pekko/japi/JavaAPI.scala

```diff
@@ -255,13 +254,16 @@ object Util {
    * Returns an immutable.Seq representing the provided array of Classes,
    * an overloading of the generic immutableSeq in Util, to accommodate for erasure.
    */
-  def immutableSeq(arr: Array[Class[_]]): immutable.Seq[Class[_]] = immutableSeq[Class[_]](arr)
+  def immutableSeq(arr: Array[Class[_]]): immutable.Seq[Class[_]] =
+    if ((arr ne null) && arr.length > 0) arr.toIndexedSeq else Nil
 
   /**
    * Turns an array into an immutable Scala sequence (by copying it).
    */
-  def immutableSeq[T](arr: Array[T]): immutable.Seq[T] =
-    if ((arr ne null) && arr.length > 0) arr.toIndexedSeq else Nil
+  @SafeVarargs
+  @varargs
```

Comment:
this is not a deprecation - please do not add a big change like this into a PR with a name like `chore: Deprecate JAPI`